packages/ui-scripts/lib/utils/handle-generate-tokens.js

```diff
@@ -22,53 +22,61 @@
  * SOFTWARE.
  */
 
-import fse from 'fs-extra'
 import path from 'path'
 import StyleDictionary from 'style-dictionary'
+import fs from 'fs'
 
 export async function handleGenerateTokens({
   themeKey,
   sourcePath,
   styleDictionarySource,
   outputPath
 }) {
-  return fse
-    .outputFile(sourcePath, JSON.stringify(styleDictionarySource))
```

Comment:
The error was likely that this was not created in some rare cases, I dont know why. I've replaced it with just native Node code